provider/snowflake.go

```diff
@@ -19,6 +20,11 @@ const (
 	sfTimestamp                     = "TIMESTAMP_NTZ"
 )
 
+var snowflakeParameterAlias = map[string]string{
```

Comment:
Why are only these two non-capital?

Reply:
These two are used in the parameters, created the struct iterator so we can add others if needed but these seem to be the most common. The godoc says they're lowercase.